File: C PROGRAMMING LANGUAGE/ASSIGNMENTS/ASSIGNMENT-8/evenodd.c
int evenodd(int[] ,int size);
void main()
{
	int arr[100],size;
	printf("enter size of array: ");
	scanf("%d",&size);
	printf("\nenter array elements: ");
	for(int i=0;i<size;i++)
		scanf("%d",&arr[i]);
	printf("\narray elements: ");
	for(int i=0;i<size;i++)
		printf("%d  ",arr[i]);
	evenodd(arr,size);
}
int evenodd(int a[] ,int n)
{
	printf("\neven numbers :");
	for(int i=0;i<n;i++)
	{
		if(a[i]%2==0)
			printf("%d  ",a[i]);
	}
	printf("\nodd numbers :");
	for(int i=0;i<n;i++)
	{
		if(a[i]%2!=0)
			printf("%d  ",a[i]);
	}
}

File: JAVA/ARRAY/EvenOdd.java
//4.	Find odd and even among the numbers.

import java.util.Scanner;

class EvenOdd
{
  public static void main(String[] args)
  {
	Scanner sc = new Scanner(System.in);
	int i=0;
	int [] arr = new int[10];
	System.out.println("enter the array elements: ");
	for(i=0;i<arr.length;i++)
	{
	   arr[i]=sc.nextInt();
 	}

	System.out.println("Array elements are: ");
	for(i=0;i<arr.length;i++)
	{
	   System.out.println(arr[i]);
 	}

	System.out.println("Even numbers: ");
	for(i=0;i<arr.length;i++)
	{
	   if(arr[i]%2==0)
		System.out.println(arr[i]);
 	}

	System.out.println("Odd numbers: ");
	for(i=0;i<arr.length;i++)
	{
	   if(arr[i]%2!=0)
		System.out.println(arr[i]);
 	}
   }
}


